ci/docker-compose.yml

```diff
@@ -22,8 +22,10 @@ services:
       # provide an id for the bridge node (default: 0)
       # each node should have a next natural number starting from 0
       - NODE_ID=0
+      # setting SKIP_AUTH to true disables the use of JWT for authentication
+      - SKIP_AUTH=true
     ports:
-      - 26658:26658
+      - 36658:26658
```

Comment:
could we revert the port change and align url in celestia-rpc tests instead?
+ bonus points for setting 46658 and NODE_ID=2 in the commented out bridge below :p
Forgive me, I swear I tried my best not commenting on this but my inner pendant won

Reply:
Reordered dockers by port and fixed the one in comment 👍